src/main/kotlin/com/karumi/katagenda/ui/ContactsListPresenter.kt

```diff
@@ -0,0 +1,68 @@
+package com.karumi.katagenda.ui
+
+import com.karumi.katagenda.common.ui.Presenter
+import com.karumi.katagenda.domain.Contact
+import com.karumi.katagenda.usecase.AddContact
+import com.karumi.katagenda.usecase.GetContacts
+
+class ContactsListPresenter(view: ContactsListPresenter.View, private val getContacts: GetContacts,
```

Comment:
Do we need to specify that the view is weak ?

Reply:
is not weak in the original kata, and reviewing  the app is not mandatory.